uint32 src_vehicle
uint32 platoon_id
float32 ipd
float32 ps
uint32[] followers